.github/.markdownlint.yaml

```diff
@@ -88,3 +88,8 @@ MD049:
 MD050:
   # Strong style should be consistent
   style: "asterisk"
+
```

Comment:
instead of this we can remove those links and mouse-over from the tables. The future tree view will make this redundant anyway.